### PR TITLE
Dockerfile + entry point for running the go-bridge-api

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@JayH5 @jpaidoussi ready for review

**Comment by smn** [comment]
@JayH5 I see you're doing some parsing of the requirements file to get a version number. This isn't going to work as this doesn't require anything other than what's already in the vumi docker image

File: go-bridge-api/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+amqp_prefetch_count: ${AMQP_PREFETCH_COUNT}
+
+redis_manager:
+  host: ${REDIS_HOST}
+  port: ${REDIS_PORT}
+  db: ${REDIS_DB}
+
+EOF
+CONFIG_FILE="/app/config.yml"
+
```
Comment (JayH5) [inline go-bridge-api/entrypoint.sh:35]: Maybe leave out the `AMQP_HOST`...`AMQP_PASSWORD` stuff as well as this twisted plugin stuff and just call the Vumi entrypoint script?

``` shell
...
export WORKER_CLASS="vumi.transports.vumi_bridge.GoConversationTransport"
export CONFIG_FILE="/app/config.yml"

exec vumi-entrypoint.sh
```

File: go-bridge-api/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ACCESS_TOKEN="${ACCESS_TOKEN:-access_token}"
+ACCOUNT_KEY="${ACCOUNT_KEY:-account_key}"
+CONVERSATION_KEY="${CONVERSATION_KEY:-conversation_key}"
+TRANSPORT_NAME="${TRANSPORT_NAME:-transport_name}"
+WEB_PATH="${WEB_PATH:-/go-bridge-api}"
+BASE_URL="${BASE_URL:-https://go.vumi.org/api/v1/go/http_api_nostream/}"
+REDIS_HOST="${REDIS_HOST:-127.0.0.1}"
+REDIS_DB="${REDIS_DB:-1}"
+REDIS_PORT="${REDIS_PORT:-6379}"
+AMQP_HOST="${AMQP_HOST:-127.0.0.1}"
```
Comment (JayH5) [inline go-bridge-api/entrypoint.sh]: So you should be able to take out these `AMQP` things now, except for `AMQP_PREFETCH_COUNT`

**Comment by JayH5** [comment]
:+1:

**Pull request merged**